### PR TITLE
Add Runstat.us

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
# runstatus cmd
image

## list page
image

## create page(s)
image

## delete page(s)
image

# Incident
look the video:
in this video now I have replaced `incident event` by `incident update`
https://sos-de-fra-1.exo.io/eip/incident2.mp4

# Maintenance
image

### CODE REVIEW

File: vendor/github.com/exoscale/egoscale/runstatus.go [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+		return nil, fmt.Errorf(`response content-type expected to be "application/json", got %q`, contentType)
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode >= 400 {
+		e := new(DNSErrorResponse)
```
Comment (greut) [inline vendor/github.com/exoscale/egoscale/runstatus.go]: meh?

Comment (pierre-emmanuelJ) [reply vendor/github.com/exoscale/egoscale/runstatus.go]: WIP sorry

Comment (pierre-emmanuelJ) [reply vendor/github.com/exoscale/egoscale/runstatus.go]: I have to implement it

File: cmd/root.go [hunk 2]
```diff
@@ -204,6 +204,7 @@ func initConfig() {
 		viper.AddConfigPath(path.Join(usr.HomeDir, ".exoscale"))
 		viper.AddConfigPath(usr.HomeDir)
 		viper.AddConfigPath(".")
+		gConfigFilePath = path.Join(gConfigFolder, fmt.Sprintf("%s.toml", defaultConfigFileName))
```
Comment (greut) [inline cmd/root.go]: it looks like #63 material.

**Comment by pierre-emmanuelJ** [comment]
[video](https://sos-de-fra-1.exo.io/eip/demo-incident.mp4)

File: cmd/runstatus_maintenance_add.go [hunk 3]
```diff
@@ -83,15 +83,39 @@ var runstatusMaintenanceAddCmd = &cobra.Command{
...
+			startDate, err = pickupDatePrompt(reader, time.Now(), "Choose start date:")
+			if err != nil {
+				return err
+			}
+		} else {
+			startDate, err = time.Parse("2006-01-02T15:04:05-0700", sDate)
+			if err != nil {
+				return err
+			}
+		}
```
Comment (greut) [inline cmd/runstatus_maintenance_add.go]: you may move the `if err != nil { return err }` after the big `if sDate`.

File: cmd/runstatus_maintenance_add.go [hunk 4]
```diff
@@ -83,15 +83,39 @@ var runstatusMaintenanceAddCmd = &cobra.Command{
...
+			endDate, err = pickupDatePrompt(reader, startDate, "Choose end date:")
+			if err != nil {
+				return err
+			}
+		} else {
+			endDate, err = time.Parse("2006-01-02T15:04:05-0700", eDate)
+			if err != nil {
+				return err
+			}
+		}
```
Comment (greut) [inline cmd/runstatus_maintenance_add.go]: same.

File: cmd/runstatus_maintenance_list.go [hunk 5]
```diff
@@ -1,17 +1,59 @@
...
+
+				for _, maintenance := range maintenances {
+					table.Append([]string{page.Subdomain, maintenance.Title, maintenance.Status})
+					page.Subdomain = ""
+				}
+
+			}
+
+			table.Render()
+			return nil
```
Comment (greut) [inline cmd/runstatus_maintenance_list.go]: else ?

File: cmd/runstatus_maintenance_add.go (hunk 3 above)
Comment (pierre-emmanuelJ) [reply cmd/runstatus_maintenance_add.go]: thx

File: cmd/runstatus_maintenance_add.go (hunk 4 above)
Comment (pierre-emmanuelJ) [reply cmd/runstatus_maintenance_add.go]: thx

File: cmd/runstatus_maintenance_add.go (hunk 3 above)
Comment (pierre-emmanuelJ) [reply cmd/runstatus_maintenance_add.go]: It's changed but github don't show it !!!

File: cmd/runstatus_maintenance_add.go (hunk 4 above)
Comment (pierre-emmanuelJ) [reply cmd/runstatus_maintenance_add.go]: same

**Comment by greut** [comment]
@pierre-emmanuelJ I'm on it...

**Comment by pierre-emmanuelJ** [comment]
very nice thanks for this new feature in egoscale 👍 🎉

File: cmd/root.go (hunk 2 above)
Comment (greut) [reply cmd/root.go]: @pierre-emmanuelJ is this still required?

Comment (pierre-emmanuelJ) [reply cmd/root.go]: It's to display more explicite error message in runstatus

Comment (greut) [reply cmd/root.go]: did you read #63 changes?

Comment (pierre-emmanuelJ) [reply cmd/root.go]: thanks you did that in root.go L207

**Pull request merged**